services/consensusalgo/benchmarkconsensus/non_leader.go

```diff
@@ -16,26 +16,17 @@ func (s *service) nonLeaderHandleCommit(blockPair *protocol.BlockPairContainer)
 
 	err := s.nonLeaderValidateBlockUnderMutex(blockPair)
 	if err != nil {
-		s.reporting.Error(err.Error()) // TODO: wrap with added context
+		s.reporting.Error("non leader failed to validate block", instrumentation.Error(err))
 		return
```

Comment:
same as above, this means the error is handled already and no other actions will be taken